mp/src/game/client/momentum/c_mom_triggers.cpp

```diff
@@ -525,4 +529,61 @@ IMPLEMENT_CLIENTCLASS_DT(C_TriggerSlide, DT_TriggerSlide, CTriggerSlide)
 RecvPropBool(RECVINFO(m_bStuckOnGround)),
 RecvPropBool(RECVINFO(m_bAllowingJump)),
 RecvPropBool(RECVINFO(m_bDisableGravity)),
-END_RECV_TABLE();
\ No newline at end of file
+END_RECV_TABLE();
+
+// ======================================
+
+LINK_ENTITY_TO_CLASS(info_teleport_destination, C_TeleportDestination);
+
+IMPLEMENT_CLIENTCLASS_DT(C_TeleportDestination, DT_TeleportDestination, CTeleportDestination)
+END_RECV_TABLE();
+
+int C_TeleportDestination::DrawModel(int flags)
+{
+    if (!mom_teledests_draw.GetBool())
+        return 1;
```

Comment:
Shouldn't this be in `ShouldDraw` ?

Reply:
For some reason this doesnt work; the behaviour here seems that if `ShouldDraw` is false once it never calls it again.